src/index.html

```diff
@@ -7,15 +7,25 @@
   <link rel="stylesheet" href="./assets/index.css">
 </head>
 
-<body>
-  
-  <div id="background">
-    <div id="landing_container"></div>
-    <div id="input_container"></div>
-    <div id="result_container"></div>
-  </div>
+  <body>
 
-<script src="./assets/index.js" charset="utf-8"></script>
-</body>
+    <div class="icon-class">
+      <img src="img/crop_icon.png" alt="image not found" height="100px" width="100px">
```

Comment:
write image as  src="./img/corp_icon.png" 
do same for all the images
and use the following images
[Landing Page (1).zip](https://github.com/CropAi/Frontend/files/4422312/Landing.Page.1.zip)

Reply:
@abhishekgupta368 please download this zip file for images and update, 
https://github.com/CropAi/Frontend/files/4422312/Landing.Page.1.zip
and use exact name of image (case-sensitive).[currently your images are like leaf.PNG and you are using pic1.png in img tag ]